src/content/docs/tutorials/json-editor/main.md

```diff
@@ -164,8 +164,8 @@ if let Event::Key(key) = event::read()? {
 
 and then match the results.
 
-Alternatively, we can set up a thread to run in the background to poll and send `Event`s (as we did
-in the "counter" tutorial). Let's keep things simple here for the sake of illustration.
```

Comment:
The reference to the counter tutorial was removed since the basic counter tutorial doesn't deal with background event polling. Perhaps this was meant to reference the async-counter tutorial, but that tutorial doesn't appear to be surfaced on the website yet.